draft-ietf-mls-protocol.md

```diff
@@ -5365,6 +5428,21 @@ Initial contents:
 | 0x0005          | MLS_256_DHKEMP521_AES256GCM_SHA512_P521             | Y | RFC XXXX |
 | 0x0006          | MLS_256_DHKEMX448_CHACHA20POLY1305_SHA512_Ed448     | Y | RFC XXXX |
 | 0x0007          | MLS_256_DHKEMP384_AES256GCM_SHA384_P384.            | Y | RFC XXXX |
+| 0x0A0A          | GREASE                                              | Y | RFC XXXX |
```

Comment:
Do we want these GREASE ciphersuites to vary the components? Ex:

GREASE_128_DHKEMX25519_AES128GCM_SHA256_Ed25519
MLS_128_GREASE_AES128GCM_SHA256_Ed25519
MLS_128_DHKEMX25519_GREASE_SHA256_Ed25519
MLS_128_DHKEMX25519_AES128GCM_GREASE_Ed25519
MLS_128_DHKEMX25519_AES128GCM_SHA256_GREASE